sagemaker-featurestore/data/feature_store_introduction_customer_updated.csv

```diff
@@ -0,0 +1,5 @@
+customer_id,city_code,state_code,country_code,email,name
+573291,1,49,2,john.lee@gmail.com,John Lee
+109382,2,40,2,olivequil@gmail.com,Olive Quil
+828400,3,31,2,liz.knee@gmail.com,Liz Knee
+124013,4,5,2,eileenbook@gmail.com,Elieen Book
```

Comment:
Spelling nit: `Elieen` -> `Eileen`?

Reply:
thx for catching it, fixed in new version.